src/Node.py

```diff
@@ -568,12 +591,17 @@ def broadcast_prepare_message(self):
         """
         Broadcast SCPPrepare message to neighboring nodes from ballot_prepare_broadcast_flags.
         """
-        if self.ballot_prepare_broadcast_flags:
-            scp_prepare_message = self.ballot_prepare_broadcast_flags.pop()
-            log.node.info('Node %s is broadcasting SCPPrepare message: %s', self.name, scp_prepare_message)
+        if len(self.ballot_prepare_broadcast_flags) == 0:
```

Comment:
this is not needed, as in the "process_prepare_ballot_message" which will be made the messages will be parsed and added to the balloting_state, and a function to retrieve a broadcast message from a quorum has also been made